jest-integration.config.js

```diff
@@ -1,10 +1,15 @@
+// Minor helper for test development. Allows for specific file testing.
+const mainTestMatch = process.env.SPECIFIC_INTEGRATION_TEST_FILE
+  ? `<rootDir>/__tests__/integration/**/${process.env.SPECIFIC_INTEGRATION_TEST_FILE}.test.js`
+  : '<rootDir>/__tests__/integration/**/*.test.js'
```

Comment:
Sugestion: take this from command line arguments so we can run specific tests with `npm run test_network_integration -- start` for instance
```suggestion
const args = process.argv.slice(6);
const testMatch = args.length == 0 ? ['<rootDir>/__tests__/integration/**/*.test.js'] : args.map(arg => `<rootDir>/__tests__/integration/**/${arg}.test.js`);
```

Obs: the `slice(6)` is because we have "node", the filename and 4 other arguments we pass to jest on package.json

Edit: actually, we can already fillter tests with the command line wihtout this.
Running `npm run test test_network_integration -- address` will match all tests on `__tests__/integration/` but will filter with the pattern `/address/`.

Reply:
I can confirm it already has this behavior. You can even inform an environment variable _and_ the command line - the later filters the first.

![Screenshot from 2022-03-31 15-15-51](https://user-images.githubusercontent.com/1299409/161123213-10a1ecf0-4eeb-4324-9a8e-7e9920c6e40f.png)


I prefer keeping it that way, so that I can use not only the `test_network_integration` script but also the `test_integration` that derives from it.